src/Header/Header.less

```diff
@@ -172,6 +172,7 @@
 					color: @moon-input-header-placeholder-color;
 					margin-top: 12px;
 					line-height: 1.25em;
+					opacity: 1;
```

Comment:
We _could_, but it doesn't hurt webkit/others, and a separate rule would just add bulk to the already very heavy CSS. I'm not concerned about hitting other browsers with this too.
The resulting selectors code would look like this:
```
.moon-header.moon-input-header .moon-input-header-input-decorator .moon-input::-webkit-placeholder {
    color: @moon-input-header-placeholder-color;
    margin-top: 12px;
    line-height: 1.25em;
}
.moon-header.moon-input-header .moon-input-header-input-decorator .moon-input::-moz-placeholder {
    color: @moon-input-header-placeholder-color;
    margin-top: 12px;
    line-height: 1.25em;
}
.moon-header.moon-input-header .moon-input-header-input-decorator .moon-input::-moz-placeholder {
    opacity: 1;
}
```